Sources/Classes/PDFViewController.swift

```diff
@@ -79,7 +79,7 @@ public final class PDFViewController: UIViewController {
     func print() {
         guard UIPrintInteractionController.isPrintingAvailable() else { return }
         guard UIPrintInteractionController.canPrintURL(document.fileURL) else { return }
-        
+        guard document.password == nil else {return }
```

Comment:
Combined init functions. Password is defaulted to String? = nil if any non nil value is provided we implement the standard unlock procedures